client/my-sites/google-my-business/select-location/index.js

```diff
@@ -0,0 +1,124 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import { connect } from 'react-redux';
+import { localize } from 'i18n-calypso';
+import page from 'page';
+import PropTypes from 'prop-types';
+import React, { Component } from 'react';
+
+/**
+ * Internal dependencies
+ */
+import Card from 'components/card';
+import CompactCard from 'components/card/compact';
+import DocumentHead from 'components/data/document-head';
+import ExternalLink from 'components/external-link';
+import { getSelectedSiteSlug } from 'state/ui/selectors';
+import GoogleMyBusinessLocation from './location';
+import HeaderCake from 'components/header-cake';
+import Main from 'components/main';
+import PageViewTracker from 'lib/analytics/page-view-tracker';
+import { recordTracksEvent } from 'state/analytics/actions';
+
+class GoogleMyBusinessSelectLocation extends Component {
+	static propTypes = {
+		locations: PropTypes.arrayOf(
```

Comment:
I wonder where you got that list of attributes from?

In the documentation there does not seem to be any photo url as you can see https://developers.google.com/my-business/reference/rest/v4/accounts.locations#Location

I thought we could have a chance of having it in the metadata object so I tried adding a logo to my gmb location but it did not show up there. In any case, the image google required there have to be pretty large (min 250px x 250px) so it won't do on a small icon next to the name and address imo.

Reply:
I took and refined this list of attributes from the prototype. I couldn't find the photo url from the Google My Business API documentation as well. The minimum size is a very good point.

Maybe @scruffian have more information about that profile photo?

Reply:
Well actually the logo I uploaded to test shows up as a small format on their gmb web console and it's not so bad:
![image](https://user-images.githubusercontent.com/230230/38237600-f910cd1c-3727-11e8-803c-b32ce5ef009d.png)
I haven't found a way to get it from the api though